test/rulesets/Base/base.jl

```diff
@@ -103,5 +103,22 @@ end
         rrule_test(identity, randn(rng), (randn(rng), randn(rng)))
         rrule_test(identity, randn(rng, 4), (randn(rng, 4), randn(rng, 4)))
     end
+
+    @testset "Constants" begin
+        function test_constant(f, x, expected)
```

Comment:
is there any other use-case for this? that would warrent it moving to `test/test_utils.jl`?

Reply:
For now I am happy to leave it here.
We can move it later.